pkg/security/probe/probe.go

```diff
@@ -1510,7 +1510,7 @@ func NewProbe(config *config.Config, statsdClient statsd.ClientInterface) (*Prob
 		},
 		manager.ConstantEditor{
 			Name:  "cgroup_activity_dumps_enabled",
-			Value: utils.BoolTouint64(config.ActivityDumpEnabled && areCGroupADsEnabled(config)),
+			Value: utils.BoolTouint64(config.ActivityDumpEnabled),
```

Comment:
Are we sure about that ?

Reply:
Doesn't this mean that we will start dumping cgroups even though it's not enabled ?

Reply:
For me it looks like a worthless double check, what is the point of enabling activity dumps without any traced cgroup slots?
I rather prefer to add a guard on traced_cgroups_count to be > 0
WDYT ?

Reply:
But `areCGroupADsEnabled` was already a guard on `traced_cgroups_count` no ?
My issue is with comm activity dumps

Reply:
> But `areCGroupADsEnabled` was already a guard on `traced_cgroups_count` no ?

Kind of yes.

> My issue is with comm activity dumps

Could you explain why please? Maybe I missed something here.

AFAIK, this constant is only checked on `should_trace_new_process_cgroup` (not `should_trace_new_process_comm`), it should be more aligned with its name now (checking only if activity dumps is enabled or not)

But the case where we enable activity dumps only to do manually dumps based on comms is something I never tested, we should add a dedicated test if it's something we want to support. Let's clarify it